### PR TITLE
Setting up the Animal class to store data

### PR DESCRIPTION
**Pull request by arianazhu**

### CODE REVIEW

File: src/main/java/com/google/sps/utils/data/Animal.java [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+    public void setFreshwaterSystem(boolean indicator) {
+        this.freshwater_system = indicator;
+    }
+
+    public void setTerrestrialSystem(boolean indicator) {
+        this.terrestrial_system = indicator;
+    }
+
+    public void setTaxonomicNotes(Stirng taxonomicnotes) {
```
Comment (maxhchen) [inline src/main/java/com/google/sps/utils/data/Animal.java]: Typo on `String`. Nitpick, but maybe change `taxonomicnotes` to `taxonomic_notes` to be consistent with parameters for other functions.

File: src/main/java/com/google/sps/utils/data/AnimalHistory.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    }
+
+    @Override
+    public int hashCode() {
+        int hash = year;
+        final int prime = 31;
+
+        hash = hash * prime + code.hashCode();
+        hash = hash * prime + category.hashCode();
+        return hash;
```
Comment (maxhchen) [inline src/main/java/com/google/sps/utils/data/AnimalHistory.java:51]: I think one of these `hash = ...` lines should be commented/removed? Or maybe combine the hash into `hash = hash * prime + code.hashCode() + category.hashCode();`?

**Review by maxhchen — APPROVED** [review]

File: src/main/java/com/google/sps/utils/data/Animal.java (hunk 1 above)
Comment (arianazhu) [reply src/main/java/com/google/sps/utils/data/Animal.java]: ooh nice catch. Yeah I copied the variable names from the IUCN API but I'll add the underscore for now and once we get the API key I'll see if it needs changes.

**Comment by fgoyal** [comment]
Can we actually change this pull request to merge into the web-scraping branch? Some of my code is still incomplete and doesn't work, and I didn't want that in dev quite yet.

Actually, I'm not sure if we can revert a merged pull request, so it should be okay!

**Pull request merged**